theunderground/movies.py

```diff
@@ -70,16 +80,19 @@ def add_movie():
                     category_id=form.category.data,
                     length=length,
                     aspect=True,
-                    genre=0,
+                    genre=genre,
                     sp_page_id=0,
                     staff=False,
                 )
 
+                if dsmovie and validation_ds:
+                    db_movie.ds_mov_id = db_movie.movie_id
+
                 db.session.add(db_movie)
                 db.session.commit()
 
                 # Now that we've inserted the movie, we can properly move it.
-                save_movie_data(db_movie.movie_id, thumbnail_data, movie_data)
+                save_movie_data(db_movie.movie_id, thumbnail_data, movie_data, dsmovie_data if dsmovie and validation_ds else None)
```

Comment:
In the else statement on line 68 you can just set `dsmovie_data` to None. It will make this line look more concise.

Reply:
that is what it does lol

Reply:
resolved